tests/unit_tests/test_game_stats_service.py

```diff
@@ -520,48 +524,62 @@ async def test_killed_acus_one_and_died(
     assert len(event_service.mock_calls) == 0
 
 
+async def test_killed_acus_three_and_died(
+    game_stats_service, achievement_service, event_service, unit_stats
+):
+    unit_stats['cdr']['kills'] = 3
+    unit_stats['cdr']['lost'] = 1
+
+    game_stats_service._killed_acus(unit_stats, False, [])
+    achievement_service.increment.assert_called_once_with(
+        ach.ACH_DONT_MESS_WITH_ME, 3, [])
```

Comment:
I personally like to align the closing brackets like this
```suggestion
        ach.ACH_DONT_MESS_WITH_ME, 3, []
    )
```